client/util/withRequests.js

```diff
@@ -0,0 +1,223 @@
+import {equal, map, path} from 'wasmuth'
```

Comment:
@adriaanwm is this functionally different in anyway? Or just renamed?

Reply:
just renamed